cmake/genmsg-extras.cmake.em

```diff
@@ -18,6 +18,9 @@ set(GENMSG_CHECK_DEPS_SCRIPT "${genmsg_DIR}/../../../@(CATKIN_PACKAGE_BIN_DESTIN
 
 include(CMakeParseArguments)
 
+# We need various macros and variables that are provided by catkin
+find_package(catkin REQUIRED)
```

Comment:
`catkin` variables are already being used in the lines above. Therefore I think this needs to be moved to line ~10.